Wanderer/www/modules/dc-list-reputations/page.html

```diff
@@ -0,0 +1,14 @@
+﻿<link rel="stylesheet" type="text/css" href="modules/dc-list-reputations/stylesheet.css">
+<div id="whole_thing">
+    <div class="reputation" data-ng-repeat="reputation in module.reputations track by $index">
+        <input type="text" data-ng-model="module.reputations[$index]"
+                           data-ng-model-options="{updateOn: 'blur default',
+                                                   debounce: {
+                                                      'default': 500,
+                                                      'blur': 0  
+                                                       }
+                                                   }"></input>
```

Comment:
Just for this one particular text input, and also maybe not at all, I haven't checked it. The problem only shows up when we're actually playing and I have my whole character loaded.

But my best guess is that putting this on all the text inputs might fix the problem for them.

I dimly remember discovering that the skill web was also somehow making it worse.